components/eamxx/src/physics/spa/eamxx_spa_process_interface.cpp

```diff
@@ -68,7 +68,7 @@ void SPA::set_grids(const std::shared_ptr<const GridsManager> grids_manager)
   // where a single column of data corresponding to the closest lat/lon pair to
   // the IOP lat/lon parameters is read from file, and that column data is mapped
   // to all columns of the IdentityRemapper source fields.
-  EKAT_REQUIRE_MSG(spa_map_file == "" or not m_iop,
+  EKAT_REQUIRE_MSG(spa_map_file == "" or spa_map_file == "None" or not m_iop,
```

Comment:
Does `None` really end up in the yaml file? It's supposed to just have an empty string if the compset doesn't require a map file...

Reply:
Yeah, a bit annoying but the way that it treats 
```
<spa_remap_file COMPSET="*.DP-EAMxx*"/>
```
is to fill in "None". Is there another way to delete the entry? Else it will get whatever is dictated by the `hgrid` value.